Project/Headers/input.h

```diff
@@ -0,0 +1,84 @@
+#ifndef INPUT_H
+#define INPUT_H
+
+#include <stm32f4xx.h>
+#include <stm32f4xx_gpio.h>
+#include <stm32f4xx_rcc.h>
+#include <stm32f4xx_exti.h>
+#include <misc.h>
+#include <stm32f4xx_syscfg.h>
+//#include <l3gd20.h>
+#include "stm32f429i_discovery.h"
+#include "stm32f429i_discovery_l3gd20.h"
+
+/*
+Button 1 - PG2
+Button 2 - PG3
+Button 3 - PD4
+*/
```

Comment:
Možeš obrisati sav ovaj kod od tipki? Ja sam noviji stavio na pull request, a i ne znam hoce se moc mergeat ako oboje imamo input.c i .h s različitim kodom - @Leonard-Blazevic kako to funkcionira?

Reply:
Ako @KarlaSalamun napravi merge nakon tebe onda ce se tvoj kod prebrisat tako da bi trebala maknit sve tvoje fileove iz svog brancha.